composer.json

```diff
@@ -21,7 +21,8 @@
         "predis/predis":           "~1.0"
     },
     "conflict": {
-        "doctrine/common": ">2.2,<2.4"
+        "doctrine/common": ">2.2,<2.4",
+        "ext-apcu":        "<4.0.8"
```

Comment:
Maybe it's best to add this kind of check at runtime in the apc cache class rather than preventing installation of the entire package?